### PR TITLE
fix example command

### PR DESCRIPTION
**Pull request by SebSept**
as discussed on #85

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,25 +45,30 @@ to extends our class.
 ```php
 <?php
 
-namespace Your\Own\Namespace;
+// psr-4 autoloader
+
+// if the command is located at src/Commands
+namespace FOP\Console\Commands; 
+// or if command is located in a subfolder
+namespace FOP\Console\Commands\Domain; // e.g. namespace FOP\Console\Commands\Configuration;xample if command
```
Comment (nenes25) [inline README.md:53]: Missing e for example ?

Comment (SebSept) [reply README.md]: fixed

**Review by nenes25 — APPROVED** [review]
Great :+1:  !

**Review by jf-viguier — APPROVED** [review]
ok thanks

**Pull request merged**